lib/dialects/mysql/data-types.js

```diff
@@ -86,7 +86,16 @@ module.exports = BaseTypes => {
     }
   }
 
-  const SUPPORTED_GEOMETRY_TYPES = ['POINT', 'LINESTRING', 'POLYGON'];
+  const SUPPORTED_GEOMETRY_TYPES = [
+    'GEOMETRY',
+    'POINT',
+    'LINESTRING',
+    'POLYGON',
+    'MULTIPOINT',
+    'MULTILINESTRING',
+    'MULTIPOLYGON',
+    'GEOMETRYCOLLECTION'
+  ];
```

Comment:
Can you please add [at least one more example here](https://github.com/sequelize/sequelize/blob/6d03c09175b2b91e19b68dd200c38c085beedc2e/lib/data-types.js#L799-L829) to showcase one of these new types?

Reply:
Will do as well, I used this branch extensively (in production) for the past year and as such I have quite a bit of code.